paladin-quests.lic

```diff
@@ -0,0 +1,105 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#cleric-quests
+=end
+
+custom_require.call(%w[common common-items common-money common-travel drinfomon equipmanager])
+
+class PaladinQuests
+  include DRC
+  include DRCI
+  include DRCM
+  include DRCT
+
+  def initialize
+    unless DRStats.paladin?
+      echo '***MUST BE A PALADIN***'
+      exit
+    end
+
+  arg_definitions = [
+      [
+        { name: 'warding', regex: /warding/i, description: 'Complete the Glyph of Warding Quest.' }
+      ],
+      [
+        { name: 'bonding', regex: /bonding/i, description: "Complete the Glyph of Bonding Quest." }
+      ],
+      [
+        { name: 'light', regex: /light/i, description: 'Complete the Glyph of Light Quest.' }
+      ],
+      [
+        { name: 'mana', regex: /mana/i, description: 'Complete the Glyph of Mana Quest.' }
+      ],
+      [
+        { name: 'ease', regex: /ease/i, description: 'Complete the Glyph of Ease Quest.' }
+      ],
+      [
+        { name: 'renewal', regex: /renewal/i, description: 'Complete the Glyph of Renewal Quest.' }
+      ]
+    ]
+
+    args = parse_args(arg_definitions)
+
+    pause
+
+    EquipmentManager.new.empty_hands
+
+    if args.warding
+      warding
+    elsif args.bonding
+      tamsine
+    elsif args.light
+      eluned
+    elsif args.mana
+      mana
+    elsif args.ease
+      ease
+    elsif args.renewal
+      renewal
+    else
+      echo '***UNKNOWN QUEST NAME***'
+    end
+  end
+
+  def warding
```

Comment:
According to https://elanthipedia.play.net/Glyph_of_Warding_walkthrough this requires a Pristine soul state. Do you want to try to check for that?

Reply:
Nah, I say let them figure out how to do it and then do it. We also don't account for money or anything like that, or them being the proper level. They need to read requirements, meet them, and then do it.